src/sentry/incidents/models.py

```diff
@@ -37,6 +39,36 @@
 from sentry.utils import metrics
 from sentry.utils.retries import TimedRetryPolicy
 
+# TODO - Move AlertsRule* and this registry to sentry.alerts
+# then rename these to be less verbose since it will be relative to the module
+alert_subscription_update_registry: dict[
```

Comment:
`alert_subscription_update` is a bit confusing 🤔 

can we call these `alert_subscription_side_effect`, `trigger`, or `callback` or something?

Reply:
yeah - i was trying to talk about what / when this should be called.. thoughts on just `alert_subscription_registry` ... `subscription_update_registry`? i was hoping to keep `registry` in there to be explicit about the pattern being used. 🤔

Reply:
`alert_subscription_callback_registry`? -- trying to figure out how to map the idea of the `AlertRule` with the `Subscription` here too 🤔

i think it'll be easier to name if it's imported from `models.alert_rule` or `models.alert` instead of incidents too. then we can use the module import name to show the mapping to the `subscription_callback_registry` which unscrambles it a bit for me.